community/kernel/src/main/java/org/neo4j/kernel/impl/store/StoreAccess.java

```diff
@@ -186,6 +201,7 @@ public CountsAccessor getCounts()
         return counts;
     }
 
+    // TODO never used???
```

Comment:
Perhaps a good opportunity to remove it then :)

Reply:
I saw a few other `TODO` in the PR as well in case it was missed.